docs/docs/basic-features/index.md

```diff
@@ -3,82 +3,125 @@ title: Feature Flags - An Overview
-
-## Flagsmith Model
-
-Here's a high level overview of the data model for Flagsmith. Fear not - it's not as complex as it looks!
-
-![Image](/img/flagsmith-model.svg)
-
-OK let's break this down.
-
-### Organisations
-
-Organisations are a way for you and other team members to manage projects and their features. Users can be members of
-multiple organisations.
-
-### Projects
-
-Projects contain one or more Environments that share a single set of Features across all of the Environments within the
-Project. Organisations can have any number of Projects.
-
-### Environments
-
-Environments are a way to separate the configuration of your features. For example, your project's Development and
-Staging environments might have a feature configured as on while it is turned off in your Production environment. A
-project can have any number of environments.
-
-### Features
-
-Features are shared between all the Environments within the Project, but their values/states can be modified between
-Environments.
-
-### Identities
-
-Identities are a particular user registration for one of your Project's environments. Registering identities within the
-client application allows you to manage features for individual users. Identity features can be overridden from your
-environment defaults. For example, joe@yourwebsite.com would be a different identity in your development environment to
-the one in production, and they can have different features enabled for each environment.
-
-For more info see [Identities](/basic-features/managing-identities).
-
-### Traits
-
-You can store any number of Traits against an Identity. Traits are key:value pairs that can store any type of data. Some
-examples of traits that you might store against an Identity might be:
-
-- The number of times the user has logged in.
-- If they have accepted the application terms and conditions.
-- Their preference for application theme.
-- If they have performed certain actions within your application.
-
-For more info see [Traits](/basic-features/managing-identities.md#identity-traits).
-
-### Segments
-
-Segments are a way to define a group of users by traits such as number of times logged in, device, location or any
-number of custom defined traits.
-
-Similarly to individual users, you will be able to override environment defaults for features. For example showing
-certain features for a "power user" segment.
-
-For more info see [Segments](/basic-features/segments.md).
+import Card from '@site/src/components/Card';
+import CardHeader from '@site/src/components/Card/CardHeader';
+import CardBody from '@site/src/components/Card/CardBody';
+
+# Feature Flags - An Overview
+
+
+Feature flags enable fine-grained control over your application's functionality. They allow you to:
+
+<div className="row">
+  <div className="col col--4">
+    <Card>
+      <CardHeader>
+        <h3>🚀 Deploy Safely</h3>
```

Comment:
This should be "Release Safely" instead. The idea is that feature flags decouple releases from deployment, so the deployment process is less important/risky when feature flags are being used.